code/modules/surgery/surgery.dm

```diff
@@ -133,54 +133,127 @@
 		return FALSE
 
 	S.begin_step(user, src, tool, target)	//start on it
+
 	var/atom/surgery_target = get_surgery_target()
 	var/success = FALSE
 
 	var/difficulty_adjust = 0
 	var/time_adjust = 0
 
+	//Clothing checks: Every lim has cloathing blockers that make surgery harder!
+	if(ishuman(owner) && ishuman(user))
+		var/mob/living/carbon/human/H = owner
+		var/mob/living/carbon/human/op = user
+		var/sanity_targeting_zone = op.targeted_organ
+
+		if(!sanity_targeting_zone)
+			sanity_targeting_zone = BP_TORSO
+
+		switch(sanity_targeting_zone)
+			if(BP_MOUTH, BP_EYES, BP_HEAD)
+				if(H.head)
+					difficulty_adjust += 30
```

Comment:
Difficulty adjust **and** time adjust is a bit much. Just a time adjust (to represent moving clothing out of the affected area) should be enough to incentivize doing things properly.